### ISSUE TITLE
click away from a selection box doesnt select the point

### ISSUE THREAD

**Issue by sixinli** [palantir/plottable#1648]
When selection is enabled, if I click (not drag) on the plot, it selects the point. However, if I have an existing selection box and click else where, it doesn't select the point I click at.

**Comment by jtlan** [comment]
That's odd. We're planning to revamp drag (#1623), so we'll note this case then. Thanks!

**Comment by jtlan** [comment]
Now that I think about it, the first click should dismiss the selection box, which is probably why it doesn't select the point. @sixinli , when you click again in the same spot, does it select the point?

**Comment by sixinli** [comment]
yes when I click again, it selects the point, and if i keep clicking, it keeps selecting the new point.

**Comment by jtlan** [comment]
This seems like the "correct" behavior, then ... ?

**Comment by sixinli** [comment]
-if i have selected a box, and click somewhere else on the plot, it deselect the selected box and doesn't select the point i click

-if i have selected a point, and click somewhere else on the plot, it deselected the selected point and selects the point i click 

i think the behavior is a bit inconsistent, but i don't feel too strong against it.

**Comment by aicioara** [comment]
This looks very similar to https://github.com/palantir/plottable/issues/1500
It is all about how the end user sets the interactions I believe. 
@sixinli can you provide a fiddle? Is this still an issue?
